tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -1828,6 +1829,32 @@ describe('runs_table', () => {
       );
     });
 
+    it('dispatches singleRunSelected on checkbox double click', async () => {
+      store.overrideSelector(getRuns, [
+        buildRun({id: 'book1', name: "The Philosopher's Stone"}),
+        buildRun({id: 'book2', name: 'The Chamber Of Secrets'}),
+        buildRun({id: 'book3', name: 'The Prisoner of Azkaban'}),
+      ]);
+
+      const fixture = createComponent(
+        ['rowling'],
+        [RunsTableColumn.CHECKBOX, RunsTableColumn.RUN_NAME],
+        true /* usePagination */
+      );
+      fixture.detectChanges();
+      await fixture.whenStable();
+
+      const rows = fixture.nativeElement.querySelectorAll(Selector.ITEM_ROW);
+      const book2 = rows[1].querySelector('mat-checkbox');
+      book2.dispatchEvent(new MouseEvent('dblclick', {relatedTarget: book2}));
+
+      expect(dispatchSpy).toHaveBeenCalledWith(
+        singleRunSelected({
+          runId: 'book2',
+        })
+      );
+    });
```

Comment:
Would it be worth to add a test that click and dblclick event are fired in such order and we get the single selection result?